### PR TITLE
fix!: swiper bump to v9.3.2 changes

### PR DESCRIPTION
**Pull request by nullxDEADBEEF**
# Description

Please include a summary of the change and which issue is fixed.

Fixes the breaking changes from bumping swiper `8.4.4` -> `9.3.2`

Fixes #

### CODE REVIEW

**Comment by nullxDEADBEEF** [comment]
Backend CI will pass once #484 has been merged

File: frontend/src/lib/components/details/recommendations.svelte [hunk 1]
```diff
@@ -14,6 +14,8 @@
 
   let loadedPage: boolean
 
+  register()
```
Comment (AndreasPB) [inline frontend/src/lib/components/details/recommendations.svelte:17]: huh, wat dis?

Comment (nullxDEADBEEF) [reply frontend/src/lib/components/details/recommendations.svelte:17]: They have removed Svelte components in v9. So we have to use the swiper element web component. Then we have to register the swiper custom elements.

See: https://swiperjs.com/element

**Comment by nullxDEADBEEF** [comment]
Recommendation carousel behaves annoyingly on mobile. It scrolls up when you try to use the carousel

**Comment by nullxDEADBEEF** [comment]
> > Recommendation carousel behaves annoyingly on mobile. It scrolls up when you try to use the carousel
> 
> Was it this way before as well?

Nope, on prod it  behaves as it should

**Comment by AndreasPB** [comment]
Also, 9.2.3 is out :DD Full-time dependency updater

**Comment by nullxDEADBEEF** [comment]
> They feel super janky on my phone (might be because of the tiny amount of side scroll 🤷‍♂️)
> 
> It does "something" when dragging, but it's only usable by clicking the arrows. It's hard to tell from the video, because I don't have an indicator of the presses, but the times it looks normal is only from pressing the arrows and not dragging
> 2023_04_20_10_44_30.mp4

Yeah that is not good at all, we need to fix that :D yay more frontend :P

**Comment by nullxDEADBEEF** [comment]
> Also, 9.2.3 is out :DD Full-time dependency updater

Will this be my new title? :P

**Comment by AndreasPB** [comment]
> > Also, 9.2.3 is out :DD Full-time dependency updater
> 
> Will this be my new title? :P

New day, nice title

https://www.youtube.com/watch?v=ANhDUcScVeM

**Comment by AndreasPB** [comment]
Sry, I merged some dependabot PRs - I hope I didn't cause a headache :D

**Comment by nullxDEADBEEF** [comment]
> Sry, I merged some dependabot PRs - I hope I didn't cause a headache :D

Nah we good :D

**Comment by nullxDEADBEEF** [comment]
#538 has been merged, closing this issue.

**Pull request closed without merging**